stac_model/input.py

```diff
@@ -86,7 +87,8 @@ class ValueScalingProcessingExpression(ProcessingExpression):
         "wrap-fill-outliers",
         "wrap-inverse-map",
     ]
-]
+    | None
+)  # noqa: E501
```

Comment:
Same here.

Reply:
what's the issue here? number of spaces? this passes ruff linting. I think we should just standardize on whatever passes ruff linting when `make lint` is run